src/builds/builds.service.ts

```diff
@@ -99,18 +99,18 @@ export class BuildsService {
     return new BuildDto(build);
   }
 
-  async stop(id: string): Promise<BuildDto> {
+  async update(id: string, body: {}): Promise<BuildDto> {
     const build = await this.prismaService.build.update({
       where: { id },
       include: {
         testRuns: true,
       },
-      data: {
-        isRunning: false,
-      },
+      data: body as Prisma.BuildUpdateInput
     });
     const buildDto = new BuildDto(build);
-    this.eventsGateway.buildFinished(buildDto);
+    if(!buildDto.isRunning){
+      this.eventsGateway.buildFinished(buildDto);
```

Comment:
could we reuse `buildUpdated` method and deprecate `buildFinished`?

Reply:
Done.